prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -266,7 +328,7 @@ class BlobAccess() : Logging {
          * If one exists for the container name and connection string, the existing one will be reused.
          * @return the blob container client
          */
-        private fun getBlobContainer(blobConnInfo: BlobContainerMetadata): BlobContainerClient {
+        fun getBlobContainer(blobConnInfo: BlobContainerMetadata): BlobContainerClient {
```

Comment:
This should probably stay private

```suggestion
        private fun getBlobContainer(blobConnInfo: BlobContainerMetadata): BlobContainerClient {
```